avocado/utils/software_manager/backends/yum.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, cmd='yum'):
             self.yum_base = yum.YumBase()
         else:
             self.yum_base = None
-            log.error("%s module for Python is required. "
+            log.debug("%s module for Python is required. "
```

Comment:
Yes, it says it's required, but still, all the methods like `install`, `remove`, etc, rely on calling the `yum` commands.  Bonus points for changing the wordings, and properly describing what's gained by using the `yum` Python module.

Reply:
That's true. I'll update the message.